src/elements/table/table-wrapper/table-wrapper.visual.spec.ts

```diff
@@ -0,0 +1,72 @@
+import { html, nothing } from 'lit';
+import { classMap } from 'lit/directives/class-map.js';
+
+import {
+  describeEach,
+  describeViewports,
+  visualRegressionFixture,
+  visualDiffDefault,
+} from '../../core/testing/private.js';
+
+import './table-wrapper.js';
+
+describe(`sbb-table-wrapper`, () => {
+  let root: HTMLElement;
+
+  const cases = {
+    negative: [false, true],
+    scrollbar: [false, true],
+  };
+
+  describeViewports({ viewports: ['zero', 'medium'] }, () => {
```

Comment:
Maybe there was a little misunderstanding, I did not mean to test striped or the size not at all. I just thought it could be in a separate test case, so it doesn't generate each combination.

Reply:
Ah you have pushed separate test in styles folder. But there the sizes are combined as well with every other case?